Benchmarks/FourParameters.cs

```diff
@@ -0,0 +1,64 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkMockNet.PCLMock;
+using FakeItEasy;
+using Moq;
+using NSubstitute;
+using Rocks;
+using System;
+
+namespace BenchmarkMockNet.Benchmarks
+{
+   public class FourParameters : MockingBenchmark
```

Comment:
Let's just start with a single benchmark with a single int parameter -- at first glance it looks like most slow down linearly with more params, but I'm not sure how the param types interact with that, re: allocations, etc.

Reply:
OK, I'll delete the two benchmarks and make one called `OneParameter`.